docs/collecting-data/collecting-from-own-applications/python-tracker/subject/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Subject"
+date: "2023-06-14"
+sidebar_position: 40
+---
+
+You may have additional information about the user (i.e. subject) performing the action or the environment in which the user has performed the action. Some of that additional data can be sent into Snowplow with each event as part of the subject class.
+
+You can create a subject like this:
+
+```python
+from snowplow_tracker import Subject
+s = Subject()
+```
+
+The Subject class has a set of `set_...()` methods to attach extra data relating to the user:
+
+- [`set_platform`](#change-the-trackers-platform-withset_platform)
+- [`set_user_id`](#set-user-id-withset_user_id)
+- [`set_screen_resolution`](#set-screen-resolution-with-set_screen_resolution)
+- [`set_viewport`](#set-viewport-dimensions-withset_viewport)
+- [`set_color_depth`](#set-color-depth-withset_color_depth)
+- [`set_timezone`](#set-timezone-withset_timezone)
+- [`set_lang`](#set-the-language-withset_lang)
+
+If you initialize a `Tracker` instance without a subject, a default `Subject` instance will be attached to the tracker. You can access that subject like this:
+
+```python
+t = Tracker(my_emitter)
+t.subject.set_platform("mob").set_user_id("user-12345").set_lang("en")
+```
```

Comment:
Recently we got the feedback from customers that our docs for the .NET tracker don't explain well how to set per-event subject. I think the problem is similar here – event level subject is explained just through a short snippet at the end of the page. Maybe we could explain right at the beginning that there are two ways to set the subject – for the whole tracker or for an event. You can do [something similar as we did in the .NET docs](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/net-tracker/subject/).